.github/workflows/push_dependabot_metadata.yml

```
@@ -7,7 +7,7 @@ on:
 
 jobs:
   send-alerts:
-    runs-on: sfdc-hk-ubuntu-latest
+    runs-on: pub-hk-ubuntu-22.04-small
```

I would recommend using the Ubuntu 24.04 runners, since they are now GA :-) (And in the future we'll be wanting to drop the old runner custom groups - there is currently very little using them since I migrated most repos over to the new runner groups recently.)

```suggestion
    runs-on: pub-hk-ubuntu-24.04-ip
```